cocos/tween/actions/action.ts

```diff
@@ -95,6 +95,8 @@ export abstract class Action {
 
     protected tag = Action.TAG_INVALID;
 
+    protected _id = -1;
```

Comment:
It is better to add comment about the valid range of id.

Reply:
Fixed.